src/lapack.jl

```diff
@@ -1,10 +1,13 @@
 module LAPACK2
 
+    using libblastrampoline_jll
     using LinearAlgebra
     using LinearAlgebra: BlasInt, chkstride1, LAPACKException
     using LinearAlgebra.BLAS: @blasfunc
     using LinearAlgebra.LAPACK: chkdiag, chkside, chkuplo
 
+    liblapack_name = libblastrampoline_jll.libblastrampoline
```

Comment:
Why couldn't these names be kept in Base btw? Then PRs like this wouldn't be needed.

Reply:
Because those names give incorrect information when alternate BLAS libraries are loaded.

Reply:
Do you mean leading those to be the same values as before or updated to refer to LBT?

Reply:
To the updated ones I suppose, I guess it is constant `"libblastrampoline.so"`, right?

Reply:
I suppose we can do that. I can make a PR reintroducing them.